binaryIntro.txt

```diff
@@ -0,0 +1,81 @@
+This is a bit
+0
+Oh it can also be
+1
+
+So yeah, a bit can be either 1 or 0
+
+Here's the number 6 stored as 8 bits:
+00000110
+
+WTF us that? Glad you asked! Let's read it right to left, well just the last 3 digits 110
+
+OK, it's not one-hundred and ten :) Let's go right to left by powers of 2, dig it:
+
+
+1     1     0
+|     |     |-- This 0 is the 0th place, so we take 2 to the 0th power, that's 1, multiplied by 0 = 0
+|     |
+|     |-- This is the 1st place, so we take 2 to the 1st power, that's 2 and multiply that by 1, that's 2
+|
+|-- This is the 2nd place, so we take 2 to the 2nd power, which is 4, multiply it by 1, that's 4
+
+Now add them up. 0 + 2 + 4 = 6
+
+WTF?
+
+No really, this is how you've been counting for years, but in powers of 10. I'll prove it.
+
+231 - ok this is two-hundred and thirty-one, let's parse it:
+
+2     3     1
+|     |     |-- This 0th place, so we take 10 to the 0th power, which is 1, multiplied by 1 = 1
+|     |
+|     |-- This is the 1st place, so we take 10 to the 1st power, which is 10, multiplied by 3 = 30
+|
+|-- This is the 2nd place, so we take 10 to the 2nd power, which is 100, multiplied by 2 = 200
+
+So it's 200 + 30 + 1
+
+
+So what happens, when we move all the number 1 place to the left. This is called shifting. Let's shift 231
+left 1 place to: 2310
+
+2     3     1      0
```

Comment:
I see where this is going, and MIND === BLOWN. In the base-10 case, we're multiplying by 10. In the base-2 case, we're multiplying by 2. No way!